test/pure/List.test.mo

```diff
@@ -1,3 +1,5 @@
+// @testmode wasi
```

Comment:
Is this just to speed up the tests? We might want to run this with the interpreter (default) to check for stack overflows.

Reply:
To me it appeared that the interpreter was more tolerant in terms stack. Maybe we should have a `// @testmode both` directive, if not already possible.